github_pr.py

```diff
@@ -42,6 +43,8 @@ def _print_pr(pr, **args):
     elif 'comments' in args and args['comments']:
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
+    elif 'filters' in args and args['filters']:
+        print "%d" % (pr['pr'].number)
```

Comment:
We talked about whether we wanted the print function to have a bunch of extra conditionals to print different pieces of info related to a PR and decided against making the user get their output a certain way or making them specify what output they wanted in addition to the filters. The logic would also possibly be duplicating in "print" all the conditionals we already had with filters. If you search for PRs from a specific owner, do you want us to print back PR - owner,PR - owner,PR - owner where owner is the same and repeated over and over? In general we want to get away from the idea that this script is just for printing and be able to store reusable logic in here that other scripts can include, then those other scripts can decide how they print/format/etc.